vcpkg/users/binarycaching.md

```diff
@@ -155,6 +156,30 @@ x-gcs,gs://<bucket-name>/my-vcpkg-cache/maybe/with`,commas/too!/,readwrite
 
 Commas (`,`) are valid as part of a object prefix in GCS. Remember to escape them in the vcpkg configuration, as shown in the previous example. GCS does not have folders (some of the GCS tools simulate folders). It is not necessary to create or otherwise manipulate the prefix used by your vcpkg cache.
 
+## <a name="gha"></a> GitHub Actions cache
+
+[!INCLUDE [experimental](../../includes/experimental.md)]
+
+```
+x-gha[,<rw>]
+```
+
+Adds the GitHub Actions cache as a provider. Vcpkg needs to be executed by a GitHub Actions runner to be able to use this binary source, as well as the URL to the cache and the access token required to read and write to the cache.
+
+### <a name="gha-quickstart"></a> Quickstart
+
+You need to export two environment variables, `ACTIONS_CACHE_URL` and `ACTIONS_RUNTIME_TOKEN`, before vcpkg is able to use the GHA cache. You can do so as follows as a step inside a job:
```

Comment:
I think you need to explain why this is being provided through a back channel like this rather than being passed through the `x-gha` configuration entry itself:

* This only makes sense to use within GitHub actions
* It never makes sense to have more than one of these configured

As an aside perhaps it should be an error if more than one is configured.

Reply:
I'm having a bit of trouble explaining this in the docs, could you maybe suggest something? Not a native speaker :pensive:

Reply:
@quyykk Here's a quick stab at trying to make the requested clarifications:

https://github.com/davisp/vcpkg-docs/commit/c1e9b88de61383f5771babbc7be17ace39b685f1

@BillyONeal For the second question about having multiple GHA providers, the original PR feedback requested that multiple GHA providers be treated as a union. That comment form @JavierMatosD is here:

https://github.com/microsoft/vcpkg-tool/pull/836#pullrequestreview-1335486289

Reply:
Thank you!